format/spec.md

```diff
@@ -985,59 +997,66 @@ Values should be stored in Parquet using the types and logical type annotations
+| **`timestamp`**    | `int64`                                                            | `TIMESTAMP_MICROS` with `adjustToUtc=false` | Stores microseconds from 1970-01-01 00:00:00.000000.                                                    |
+| **`timestamptz`**  | `int64`                                                            | `TIMESTAMP_MICROS` with `adjustToUtc=true`  | Stores microseconds from 1970-01-01 00:00:00.000000 UTC.                                                |
+| **`timestamp_ns`**   | `int64`                                                          | `TIMESTAMP_NANOS` with `adjustToUtc=false`  | Stores nanoseconds from 1970-01-01 00:00:00.000000000.                                                  |
+| **`timestamptz_ns`** | `int64`                                                          | `TIMESTAMP_NANOS` with `adjustToUtc=true`   | Stores nanoseconds from 1970-01-01 00:00:00.000000000 UTC.                                              |
+| **`string`**       | `binary`                                                           | `UTF8`                                      | Encoding must be UTF-8.                                                                                 |
+| **`uuid`**         | `fixed_len_byte_array[16]`                                         | `UUID`                                      |                                                                                                         |
+| **`fixed(L)`**     | `fixed_len_byte_array[L]`                                          |                                             |                                                                                                         |
+| **`binary`**       | `binary`                                                           |                                             |                                                                                                         |
+| **`struct`**       | `group`                                                            |                                             |                                                                                                         |
+| **`list`**         | `3-level list`                                                     | `LIST`                                      | See Parquet docs for 3-level representation.                                                            |
+| **`map`**          | `3-level map`                                                      | `MAP`                                       | See Parquet docs for 3-level representation.                                                            |
+| **`geometry`**     | `binary`                                                           | `GEOMETRY`                                  | WKB format, see Appendix G. Logical type annotation optional for supported Parquet format versions [1]. |
```

Comment:
I could add this section later too, once its implemented (same for ORC below)

Reply:
```[Appendix G](#appendix-g)```